<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Start Date</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>End Date</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDateTimeEdit" name="startdateedit">
               <property name="currentSection">
                <enum>QDateTimeEdit::DaySection</enum>
               </property>
               <property name="displayFormat">
                <string notr="true">yyyy-MM-dd hh:mm:ss</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateTimeEdit" name="enddateedit">
               <property name="currentSection">
                <enum>QDateTimeEdit::DaySection</enum>
               </property>
               <property name="displayFormat">
                <string notr="true">yyyy-MM-dd hh:mm:ss</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="load_btn">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LegendTableView" name="legendtable"/>
           </item>
           <item>
            <widget class="QCheckBox" name="pressure_check">
             <property name="text">
              <string>Show Pressure</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="updatetime_check">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Update Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="updatetime_spin">
               <property name="minimum">
                <double>10.000000000000000</double>
               </property>
               <property name="maximum">
                <double>60.000000000000000</double>
               </property>
               <property name="value">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="GraphicsLayoutWidget" name="graphics_layout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>LegendTableView</class>
   <extends>QTableView</extends>
   <header>qt_extensions.legendtable</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>graphics_layout</tabstop>
  <tabstop>load_btn</tabstop>
  <tabstop>legendtable</tabstop>
  <tabstop>pressure_check</tabstop>
  <tabstop>updatetime_check</tabstop>
  <tabstop>updatetime_spin</tabstop>
  <tabstop>startdateedit</tabstop>
  <tabstop>enddateedit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
